src/py/flwr/simulation/ray_transport/ray_client_proxy.py

```diff
@@ -115,6 +116,117 @@ def reconnect(
         return common.DisconnectRes(reason="")  # Nothing to do here (yet)
 
 
+class RayActorClientProxy(ClientProxy):
+    """Flower client proxy which delegates work using Ray."""
+
+    def __init__(
+        self, client_fn: ClientFn, cid: str, actor_pool: VirtualClientEngineActorPool
+    ):
+        super().__init__(cid)
+        self.client_fn = client_fn
+        self.actor_pool = actor_pool
+
+    def _submit_job(self, job: Callable, timeout: Optional[float]):
```

Comment:
How about this naming for consistency?
```suggestion
    def _submit_job(self, job_fn: Callable, timeout: Optional[float]):
```